### DISCUSSION TITLE
tree support blocker

### DISCUSSION THREAD

**Discussion by Mario-vminion** [OrcaSlicer/OrcaSlicer#202] [Q&A]
maybe you can remove this question ... i have found the solution...

**Comment by Mario-vminion** [comment]
i have found the solution ...

**Reply by SoftFever** [reply]
Hi @Mario-vminion ,

Glad you found the solution.

It's encouraged to keep the question and the solution. It could be beneficial for others in the future.

**Reply by Mario-vminion** [reply]
I found the solution on the youtube from bambulab that its better to use the bucketicon for blocking the tree support and it worked
